visualization/app/codeCharta/ui/artificialIntelligence/artificialIntelligence.component.html

```diff
@@ -1,121 +1,138 @@
-<md-menu class="artificial-intelligence-component">
-	<md-button class="ai-button" ng-click="$mdMenu.open($event);hideBadge = true;" title="Open Suspicious Metrics Panel">
-		<i class="fa fa-cogs"></i>
-		<span
-			id="ai-badge"
-			ng-show="
-				!hideBadge &&
-				($ctrl._viewModel.riskProfile !== undefined ||
-					$ctrl._viewModel.suspiciousMetricSuggestionLinks.length ||
-					$ctrl._viewModel.unsuspiciousMetrics.length)
-			"
+<div class="artificial-intelligence-component">
+	<md-menu>
+		<md-button
+			class="suspicious-metrics-button"
+			ng-click="$mdMenu.open($event);hideBadge = true;"
+			title="Open Suspicious Metrics Panel"
 		>
-			{{
-				($ctrl._viewModel.riskProfile !== undefined ? 1 : 0) +
-					($ctrl._viewModel.suspiciousMetricSuggestionLinks.length ? 1 : 0) +
-					($ctrl._viewModel.unsuspiciousMetrics.length ? 1 : 0)
-			}}
-		</span>
-	</md-button>
+			<i class="fa fa-exclamation-triangle"></i>
+			<span
+				id="ai-badge"
+				ng-show="
+					!hideBadge &&
+					($ctrl._viewModel.riskProfile !== undefined ||
+						$ctrl._viewModel.suspiciousMetricSuggestionLinks.length ||
+						$ctrl._viewModel.unsuspiciousMetrics.length)
+				"
+			>
+				{{
+					($ctrl._viewModel.riskProfile !== undefined ? 1 : 0) +
+						($ctrl._viewModel.suspiciousMetricSuggestionLinks.length ? 1 : 0) +
+						($ctrl._viewModel.unsuspiciousMetrics.length ? 1 : 0)
+				}}
+			</span>
+		</md-button>
 
-	<md-menu-content class="ai-drop-down" width="7">
-		<md-menu-item class="ai-item" ng-show="$ctrl._viewModel.unsuspiciousMetrics.length"
-			><md-button ng-disabled="true">
-				<span ng-show="$ctrl._viewModel.analyzedProgrammingLanguage.length"
-					>Unsuspicious Metrics in {{ $ctrl._viewModel.analyzedProgrammingLanguage }} code</span
-				>
-				<span ng-show="!$ctrl._viewModel.analyzedProgrammingLanguage.length">Unsuspicious Metrics</span>
-			</md-button></md-menu-item
-		>
+		<md-menu-content class="ai-drop-down">
+			<md-menu-item class="ai-item" ng-show="$ctrl._viewModel.unsuspiciousMetrics.length">
+				<md-button ng-disabled="true">
```

Comment:
Because there is a weird issue with md where the whole menu would be wrongly positioned(the whole menu would start above the button) That solution what already used before we just copied it. If there is another cleaner way, let us know